packages/account/src/Components/icon-message-list/icon-message-list.tsx

```diff
@@ -17,7 +32,7 @@ const ListItem = ({ text }) => (
     </div>
 );
 
-const IconMessageList = ({ className, icon, message, message_list, onContinue }) => {
+const IconMessageList = ({ className, icon, message, message_list, onContinue }: TIconMessageList) => {
```

Comment:
All props of thTIconMessageList seem to be optional. I think you should make them as optional